### ISSUE TITLE
Linking Nav Terminal and a Docking Port with an interrupted wire still shows Dock button

### ISSUE THREAD

**Issue by luctius** [Regalis11/Barotrauma#8143]
Not sure if this is a known issue, but not one I could find.

When linking a Nav Terminal and a docked Docking Port with an interrupted wire (for example via a relay in the off state) still shows Dock button on the nav terminal UI. It is not usable though.

To Reproduce, use the following test bench and press "signal out" to disable the relay.
[testbench.sub.zip](https://github.com/Regalis11/Barotrauma/files/7974365/testbench.sub.zip)

**Comment by MaaiG** [comment]
Confirm. I’ve dealt with this problem before, but I didn’t make a issue report.

I’m not sure I fully understand the problem. In my case, the docking button was displayed on the terminal even when the docking port with the shuttle is not connected to the terminal (wires hung in the air or were removed)

My shuttles are docked from the main boat by an external button or from the shuttle terminal. I don’t remember how, but I got out of that position by changing the connection pattern. It seems to be related to the type of signal that arrives at the connection point of the docking port.
